src/styles/stars.css

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(../images/star.svg) no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: 50%;
```

Comment:
you can add this to the `background` prop.
`background: 50% no-repeat url(../images/star.svg);`